docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md

```diff
@@ -0,0 +1,74 @@
+---
+id: mdp_application
+title: MDP Application
+description: Behaviour Twin MDP Application Success Story
+---
+
+![Model Based Development and Data Processing Kit banner](@site/static/img/kit-icons/behaviour-mdp-kit-icon.svg)
+
+### Model Based Development and Data Processing Kit
+
+## Introduction to the Use Case
+
+The Behaviour Twin MDP (Model based development and Data Processing) Application was developed within the framework of the Catena-X initiative, to enable collaborative utilization of behavioural digital twins in the automotive industry. This innovative application is poised to define the workflow and the tool chain for the model-based system development and data processing, leveraging Catena-X technology and collaborative principles. The mission of the Behaviour Twin MDP Application is to provide a comprehensive framework for the selection, integration, and utilization of behavioural digital twins. Through adherence to Catena-X standards and principles, we strive to ensure secure data exchange, interoperability, and compatibility, laying the foundation for seamless collaboration and efficiency in innovation.
+
+The objective is to enhance model-based design and data processing using models coming from component suppliers (for example: the OEM gets the models of the gear box, the brakes and the battery from the respective Tier1s), by facilitating model exchange between Tier1s and OEM or between Tier2s and Tier1. This is done using the industry standard FMU (which is a black box model, so the intellectual property is protected) and performing the exchange over EDC. The MDP KIT defines the functionality of the software applications needed for the MDP use cases, that is to combine the models into a simulation architecture and perform parametric studies, but also to do field data processing based on evaluation models of the component suppliers.
```

Comment:
I would suggest to link the FMU standard.

Reply:
Link has been added to the Catena-X standard CX-0102